docs/monitors/vsphere.md

```diff
@@ -360,6 +361,7 @@ dimensions may be specific to certain metrics.
 | `guest_family` | For virtual machine metrics, the guest operating system family for the virtual machine to which this metric pertains. For example 'linuxGuest'. |
 | `guest_fullname` | For virtual machine metrics, the full name of the guest operating system for the virtual machine to which the metric pertains. For example 'Windows 2000 Professional'. |
 | `guest_id` | For virtual machine metrics, the guest identifier of the virtual machine to which the metric pertains. |
+| `host` | For virtual machine metrics, the host dimension will be set to the VM IP instead of the running agent hostname. This behavior is configurable, check the `vmHostDimension` option for details. |
```

Comment:
```suggestion
| `host` | For virtual machine metrics, the host dimension will be set to the VM IP instead of the running agent hostname. This behavior is configurable; check the `vmHostDimension` option for details. |
```